src/blog/2024/04/node-red-dashboard-2-layout-navigation-styling.md

```diff
@@ -0,0 +1,186 @@
+
+#### Collapsing
+
+This is the default sidebar, when it's opened, the page content adjusts to the width of the sidebar.
+
+!["Image showing 'Collapsing' sidebar"](./images/node-red-dashboard-2-layout-navigation-styling-collapsing-sidebar.gif "Image showing Collapsing sidebar"){data-zoomable}
+
+You can see in the image above how the page content automatically adjusts when the sidebar is opened.
+
+#### Fixed
+
+In this type, the sidebar is always visible and fixed on the left side, and the top menu icon is hidden. The page content adjusts to the width of the sidebar.
+
+!["Image showing 'Fixed' sidebar"](./images/node-red-dashboard-2-layout-navigation-styling-fixed-layout.png "Image showing Fixed sidebar"){data-zoomable}
+
+#### Collapse to icon
+
+This type of sidebar is similar to the collapsible one, but when the sidebar is collapsed, you can still navigate through different pages as the page icons become visible.
+
+!["Image showing 'Collapse to icon' sidebar"](./images/node-red-dashboard-2-layout-navigation-styling-collaps-to-icon-sidebar.gif "Image showing Collapse to icon sidebar"){data-zoomable}
+
+#### Apear over content 
+
+When the sidebar is opened, the page is partially covered by a transparent layer, and the sidebar appears on top of this layer
+
+!["Image showing 'Apear over content' sidebar"](./images/node-red-dashboard-2-layout-navigation-styling-appear-over-content.gif "screenshot displaying searching for botFather bot for creating custom bot"){data-zoomable}
+
+In this type of sidebar, you can notice how the sidebar opens without affecting the width of the page content
+
+#### Always hide 
+
+In this type, the sidebar is always hidden, and navigation between different pages can be achieved using the ui-control widget.
+
+!["screenshot displaying searching for botFather bot for creating custom bot"](./images/node-red-dashboard-2-layout-navigation-styling-always-hidden.png "screenshot displaying searching for botFather bot for creating custom bot"){data-zoomable}
+
+### Setting sidebar 
+
+1. Navigate to the Dashboard 2.0 sidebar.
+2. Click on the "Edit Settings" button located at the top left side of the Dashboard 2.0 sidebar.
+3. Select your preferred sidebar style from the "Style" field in the sidebar options section.
+
+!["Image showing process of changing sidebar style"](./images/node-red-dashboard-2-layout-navigation-styling-setting-sidebar.gif "Image showing process of changing sidebar style"){data-zoomable}
+
+## Styling your Dashboard 2.0 elements
+ 
+In Dashboard 2.0, we can add classes to almost all widgets, pages, and groups and style them using CSS. Applying styles to these classes will inject the CSS inline, giving them an inline scope. This means the specified styles will take precedence over any existing CSS rules.
+
+### Adding classes
+
+1. To add classes to your widget, page, or group, you'll need to open its configuration
+2. Find the 'Class' field and enter your class.
```

Comment:
Add screenshot here add a red rectangle or something around the "class" field row